guides/release/in-depth-topics/native-classes-in-depth.md

```diff
@@ -485,7 +485,7 @@ let matt = new Person();
 console.log(matt.name); // Matthew Beale
 ```
 
-Ember provides a number of decorators, such as the `@tracked` decorator, that
+Ember provides a number of decorators, such as the [`@tracked` decorator](https://api.emberjs.com/ember/3.27/modules/@ember%2Fobject#functions), that
```

Comment:
Hi, @rajakvk. To confirm, were you wanting to provide (1) a link to the page for the `@tracked` decorator, or (2) a link to a list of possible decorators?

If choice (1), I suggest that we redirect the user to the actual page for `@tracked`, like so:

```suggestion
Ember provides a number of decorators, such as the [`@tracked` decorator](https://api.emberjs.com/ember/release/functions/@glimmer%2Ftracking/tracked), that
```

Notice the use of `/release` instead of `/3.27`. This way, future versions of Ember Guides will always point to the latest version of the page in the API Guides.

If choice (2), I think the link you suggested is okay, but not the best (as the page lacks context). Regardless, I'd add the hyperlink to the phrase "a number of decorators" instead:

```suggestion
Ember provides [a number of decorators](https://api.emberjs.com/ember/release/modules/@ember%2Fobject#functions), such as the `@tracked` decorator, that
```

Hope the suggestions made sense.

Reply:
@ijlee2 Updated both links as suggested. Code pushed. Please review.